lib/addict/controllers/addict_controller.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Addict.AddictController do
```

Comment:
Can we rename this to `Addict.Controller`? I had this kind of naming before but I don't like it much :)